glmtools/glm.py

```diff
@@ -53,5 +61,33 @@ def t_test(c, X, B, sigma_2, df):
     p : array shape (V,)
         two-tailed probability value for each t statistic.
     """
-    # Your code code here
-    return
+
+    #get t statistic
+    c_b_cov = c.dot(npl.pinv(X.T.dot(X))).dot(c)
+    t = c.dot(B) / np.sqrt(sigma_2 * c_b_cov)
+
+    #get two-tailed p-value for t statistic
+    t_dist = scipy.stats.t(df=df)
+    if t > 0:
+        p = 2 * (1 - t_dist.cdf(t))
+    else:
+        p = 2 * (t_dist.cdf(t))
+    return t, p
+
+def test_glm_t_test():
```

Comment:
I was expecting you to put the tests in the `test_glm.py`, `test_glm4d.py` files...